src/Event.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _Event_H_
+#define _Event_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef struct Event* EventPtr;
+
+void Events_system_init(void);
+void Event_invoke(EventPtr event);
+uint32_t Event_addListener(EventPtr event, void (*ptr)(void));
+uint32_t Event_removeListener(EventPtr event, void (*ptr)(void));
+void Event_wipeAllListener(EventPtr event);
+uint32_t Event_internal_getElementIndex(EventPtr event, void (*ptr)(void));
```

Comment:
If the `internal` here implies that this function is only used inside of the Event module then you should not expose it here in the header and instead declare it as a static function at the top of the .c file

Reply:
I've made this static now. Do we have a common practice for naming static functions? I'm guessing '_internal' could be overexplaining something which clearly has the static keyword